simphony/io/tests/test_h5_materials.py

```diff
@@ -0,0 +1,230 @@
+import os
+import tempfile
+import shutil
+import unittest
+from functools import partial
+
+import uuid
+import tables
+
+from simphony.cuds.material import Material
+from simphony.io.h5_cuds import H5CUDS
+from simphony.io.h5_materials import H5Materials
+from simphony.core.cuba import CUBA
+from simphony.core.data_container import DataContainer
+from simphony.testing.utils import compare_materials
+
+
+class TestH5MaterialsAddGetMaterial(unittest.TestCase):
+
```

Comment:
This might be named `TestH5MaterialsMethods`, unless we are going to define multiple classes for testing groups of material methods. Just a suggestion though.